cmd/remoterun/deploy.go

```diff
@@ -135,6 +137,14 @@ It is important that this command does the minimum and must not do calculations
 				runner: runner,
 			}
 
+			sshAgentHostname := os.Getenv(constants.OktetoSshAgentHostnameEnvVar)
+			sshAgentPort := os.Getenv(constants.OktetoSshAgentPortEnvVar)
```

Comment:
Should we validate that when one is set the other one is also set?

Reply:
I don't think there could be possible that case. If one is defined, the others are too, but added them to the check just incase